docs/dev.md

```diff
@@ -189,6 +189,70 @@ docker-compose exec app python manage.py test marcha.path.to.module.Class
 docker-compose exec app python manage.py test marcha.path.to.module.Class.method
 ```
 
+### Visual regression testing
+
+Visual regression testing is available through [jest-image-snapshot](https://github.com/americanexpress/jest-image-snapshot)
+and [jsdom-screenshot](https://github.com/dferber90/jsdom-screenshot).
+Two wrappers have been added in `src/frontend/utils/tests/imageSnapshot.ts`
+
+#### imageSnapshot(width?: number, height?: number)
+
+Simply takes a screenshot and compares it to the previous one if it exists.
+**Current test states will be kept.**
+
+#### renderImageSnapshot(component: JSX.Element, width?: number, height?: number)
+
+Renders passed component wrapped with wrapInIntlProvider, wrapInRouter and Grommet.
+GlobalStyles are added to.
+imageSnapshot is called afterwards.
+**Current test states will be lost.**
```

Comment:
This can be a problem If you want to take snapshot at different states of your component.

Reply:
Indeed, that's why I added `imageSnapshot` which only takes screenshot and doesn't render the component.

Reply:
```suggestion
**This helper will destroy your current DOM generated by you test and will replace it by the one generated in this function.**
```